csm_web/frontend/static/frontend/css/policies-styles.css

```diff
@@ -0,0 +1,34 @@
+.policy-sections {
+	margin: auto;
+	width: 60%;
+	font-size: 1.05em;
+	line-height: 1.5em;
+}
+
+.policy-sections div {
+	padding-bottom: 0.5rem;
+}
+
+.policy-sections h1 {
+	color: var(--csm-green);
+	/*
+		CSM green, but IDK if I can import vars from other CSS files lol.
+		OG color was: #646464 (feel free to change back, just trying out green!)
+	*/
```

Comment:
This works perfectly fine! You can remove the comment

Reply:
epico - comment removed